### PR TITLE
chore: add market_info test

### PR DESCRIPTION
**Pull request by radoslaw-szpiech**

### CODE REVIEW

File: tests/market/test_market_info.py [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+    fields = [
+        ["Market Name", "BTC:DAI_2023"],
+        ["Code", "BTC:DAI_2023"],
+        ["Product Type", "Future"],
+        ["Quote Name", "BTC"],
+    ]
+    validate_info_section(page, fields)
+
+
+# @pytest.mark.skip("oracle test to be fixed")
```
Comment (bwallacee) [inline tests/market/test_market_info.py:121]: Need removing?

File: tests/market/test_market_info.py [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+    page.pause()
+    expect(
+        page.locator('[data-state="open"]').get_by_test_id("accordion-content")
+    ).to_contain_text("No oracle proof for settlement data")
+    expect(page.get_by_test_id("oracle-spec-links")).to_have_text(
+        "View settlement data specification"
+    )
+    # expect(page.get_by_test_id("oracle-spec-links")).to_have_attribute(
+    #     "href", re.compile(rf'(\/oracles\/{vega.find_market_id("BTC:DAI_2023")})')
+    # )
```
Comment (bwallacee) [inline tests/market/test_market_info.py:134]: Needs uncommenting?

File: tests/market/test_market_info.py [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+        ["Quote Name", "BTC"],
+    ]
+    validate_info_section(page, fields)
+
+
+# @pytest.mark.skip("oracle test to be fixed")
+def test_market_info_oracle(page: Page, vega: VegaService):
+    # 6002-MDET-203
+    page.get_by_test_id(market_title_test_id).get_by_text("Oracle").click()
+    page.pause()
```
Comment (bwallacee) [inline tests/market/test_market_info.py:125]: need to be removed

**Review by bwallacee — COMMENTED** [review]
Will these tests will only pass if we run as a group?

File: tests/market/test_market_info.py (hunk 1 above)
Comment (radoslaw-szpiech) [reply tests/market/test_market_info.py:121]: will be fixed in next PR with oracle fixed in console

File: tests/market/test_market_info.py (hunk 2 above)
Comment (radoslaw-szpiech) [reply tests/market/test_market_info.py:134]: will be fixed in next PR with oracle fixed in console

File: tests/market/test_market_info.py (hunk 3 above)
Comment (radoslaw-szpiech) [reply tests/market/test_market_info.py:125]: will be fixed in next PR with oracle fixed in console

**Review by bwallacee — APPROVED** [review]

File: tests/market/test_market_info.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (radoslaw-szpiech) [reply tests/market/test_market_info.py:125]: Oh yeah. Test is skipped anyway and I will fix that in next PR

**Pull request merged**